src/components/GiveProject/ProjectCard.tsx

```diff
@@ -254,7 +206,11 @@ export default function ProjectCard({ project, mode }: ProjectDetailsProps) {
 
   const renderGoalCompletionDetailed = (): JSX.Element => {
     const goalProgress = parseFloat(getGoalCompletion()) > 100 ? 100 : parseFloat(getGoalCompletion());
-    const formattedTotalDonated = new BigNumber(parseFloat(totalDonated).toFixed(2)).toFormat();
+    const formattedTotalDonated = !totalDonated
+      ? new BigNumber("0")
+      : new BigNumber(totalDonated).toNumber().toFixed(2);
```

Comment:
You should not need to convert the BigNumber to a number in order to use toFixed().